### PR TITLE
[Request][8.16] Update nav references for all "Investigation tools" topics

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5978

Previews:

- [Timeline](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/timelines-ui.html)
- [Timeline templates](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/timeline-templates-ui.html)
- [Visual event analyzer](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/visual-event-analyzer.html)
- [Session View](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/session-view.html)
- [Run Osquery from investigation guides](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/invest-guide-run-osquery.html)
- [Indicators of compromise](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/indicators-of-compromise.html)
- [Open and manage cases](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/cases-open-manage.html)
- [Configure case settings](https://security-docs_bk_6000.docs-preview.app.elstc.co/guide/en/security/master/cases-manage-settings.html)

### CODE REVIEW

**Comment by nastasha-solomon** [comment]
Reverting this to draft to make a few more changes.

File: docs/osquery/invest-guide-run-osquery.asciidoc [hunk 1]
```diff
@@ -39,6 +40,8 @@ image::images/setup-osquery-investigation-guide.png[width=70%][height=70%][Shows
 [[run-live-queries-ig]]
 === Run live queries from an investigation guide
 
+. Go to the *Rules* page. To access it, find **Detection rules (SIEM)** in the main menu or look for “Detection rules (SIEM)” using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```
Comment (natasha-moore-elastic) [inline docs/osquery/invest-guide-run-osquery.asciidoc:43]:
```suggestion
. Go to the *Rules* page. To access it, find **Detection rules (SIEM)** in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. 
```

**Review by natasha-moore-elastic — COMMENTED** [review]
Just a couple of minor suggestions, otherwise LGTM!

File: docs/events/timeline-templates.asciidoc [hunk 2]
```diff
@@ -74,7 +74,8 @@ filter (refer to <<pivot>>).
 . Choose one of the following:
 +
 
-** Go to **Timelines** → **Templates**, then click **Create new Timeline template**.
+** Find **Timelines** in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+** Select the **Templates** tab, then click **Create new Timeline template**.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/events/timeline-templates.asciidoc]: Good catch, thanks! Fixed in [107cca7](https://github.com/elastic/security-docs/pull/6000/commits/107cca71c8fde6f77895116eeced46094f42c70c).

**Review by natasha-moore-elastic — APPROVED** [review]

**Pull request merged**